doc/en/user/source/production/config.rst

```diff
@@ -3,121 +3,196 @@
-----------------------------------
+''''''''''''''''''''''''''''''''''
 
-GeoServer includes support for WFS-T (transactions) by default, which lets users modify your data.
+GeoServer includes support for WFS-T (transactions) which lets users modify your data.
 
 If you don't want your database modified, you can turn off transactions in the :ref:`services_webadmin_wfs`. Set the :guilabel:`Service Level` to ``Basic``. For extra security, we recommend any database access use datastore credentials providing read-only permissions. This will eliminate the possibility of a SQL injection (though GeoServer is generally not vulnerable to that sort of attack).
 
 If you would like some users to be able to modify data, set the service level :guilabel:`Service Level` to ``Transactional`` (or ``Complete``) and use :ref:`security_service` to limit access to the `WFS.Transaction` operation.
 
 If you would like some users to be able to modify some but not all of your data, set the :guilabel:`Service Level` to ``Transactional`` (or ``Complete``), and use :ref:`security_layer` to limit write access to specific layers. Data security can be used to allow write access based on workspace, datastore, or layer security.
 
-Cache your data
----------------
+GeoServer Data Admin  Guidance
+------------------------------
 
-Server-side caching of WMS tiles is the best way to increase performance.  In caching, pre-rendered tiles will be saved, eliminating the need for redundant WMS calls.  There are several ways to set up WMS caching for GeoServer.  GeoWebCache is the simplest method, as it comes bundled with GeoServer.  (See the section on :ref:`gwc` for more details.)  Another option is `TileCache <http://tilecache.org>`__.
+Establishing a data administrator user is a recommended configuration to priviledged users with limted access to the Admin Console to manage the publication of information, but are not intended to be trusted with the full global settings and system integration controls.
 
-You can also use a more generic non-spatial caching system, such as `OSCache <http://www.opensymphony.com/oscache/>`__ (an embedded cache service) or `Squid <http://www.squid-cache.org>`__ (a web cache proxy).
+1. Create a role to be used for data administration.
 
-Caching is also possible for WFS layers, in a very limited fashion. For DataStores that don't have a quick way to determine feature counts (e.g. shapefiles), enabling caching can prevent querying a store twice during a single request. To enable caching, set the Java system property ``org.geoserver.wfs.getfeature.cachelimit`` to a positive integer. Any data sets that are smaller than the cache limit will be cached for the duration of a request, which will prevent the dataset from being queried a second time for the feature count. Note that this may adversely affect some types of DataStores, as it bypasses any feature count optimizations that may exist.
+2. Optional: Assign this role ot to user to associate GROUP_ADMIN.
```

Comment:
Spelling/grammar: ot, and check the rest of the sentence

Reply:
Ended up with:

> 4. Recommendation: The combination of data admin permission for a workspace and GROUP_ADMIN access provides a good combination for an individual responsible for a workspace. This provides the ability to manage and control access to the data products in a workspace.